src/main/java/com/ericsson/eiffelcommons/jenkins/JenkinsManager.java

```diff
@@ -463,15 +467,15 @@ private void jobNameTokenValidation(String jobName, String jobToken) throws Exce
         if (jobName == null || jobName.isEmpty()) {
             throw new Exception("Cannot trigger a jenkins job without a job name.");
         }
-        if (jobToken == null) {
+        if (jobToken == null || jobToken.isEmpty()) {
```

Comment:
I think a token may be empty if you activate "trigger by token" but leaves it empty.

Reply:
Aha, I will remove this check then.